eq-author/src/components/NavigationSidebar/SectionNavItem.js

```diff
@@ -1,28 +1,36 @@
+import gql from "graphql-tag";
 import React from "react";
 import styled from "styled-components";
+import { withRouter } from "react-router-dom";
 
+import CustomPropTypes from "custom-prop-types";
 import PageNav from "components/NavigationSidebar/PageNav";
 import NavLink from "./NavLink";
-import CustomPropTypes from "custom-prop-types";
-import { buildSectionPath } from "utils/UrlUtils";
+import { buildQuestionnairePath } from "utils/UrlUtils";
 import SectionIcon from "./icon-section.svg?inline";
-import gql from "graphql-tag";
 
 const StyledSectionNavItem = styled.li`
   display: block;
 `;
 
-class SectionNavItem extends React.Component {
+export class UnwrappedSectionNavItem extends React.Component {
   static propTypes = {
     questionnaire: CustomPropTypes.questionnaire,
-    section: CustomPropTypes.section.isRequired
+    section: CustomPropTypes.section.isRequired,
+    match: CustomPropTypes.match
   };
 
   render() {
-    const { questionnaire, section, ...otherProps } = this.props;
-    const url = buildSectionPath({
+    const { questionnaire, section, match, ...otherProps } = this.props;
+    let tab = match.params.tab;
+    if (tab === "routing") {
+      tab = "design";
```

Comment:
This feels weird. I trust there's no better way?

Reply:
I pushed this into the Routing page route itself so it will ensure that it only gets pages.